exp-player/package.json

```diff
@@ -57,7 +57,8 @@
     "broccoli-merge-trees": "^1.1.1",
     "ember-cli-babel": "^5.1.5",
     "ember-cli-htmlbars": "0.7.4",
-    "ember-cp-validations": "2.9.2"
+    "ember-cp-validations": "2.9.2",
```

Comment:
The current versions are 2.9.7 (point release) or 3.1.0. Can you please clarify why this PR pins to an older version instead?